autoload/SpaceVim/layers/lang/javascript.vim

```diff
@@ -126,4 +126,10 @@ function! s:tern_go_to_def() abort
   endif
 endfunction
 
+function! s:checktime_if_javascript() abort
+  if (match(&filetype, '^javascript') == 0)
+    checktime
```

Comment:
why do not use `if &ft ==# 'javascript'`

Reply:
Because there are filetypes like `javascript.jsx`... (lol)

Reply:
but use two steps is slow, we can change it to `if &ft =~# '^javascript'`

Reply:
@wsdjeg Great. I fix it in https://github.com/SpaceVim/SpaceVim/pull/1655